packages/config/config/devices/0x027a/zen16.json

```diff
@@ -233,8 +227,32 @@
 		},
 		{
 			"#": "6",
-			"label": "Auto Turn-Off Timer for Relay 1",
-			"description": "Sets the time after which you want relay 1 to automatically turn off once it has been turned on.",
+			"$if": "firmwareVersion < 1.1",
+			"label": "Relay 1: Auto Turn-Off Timer",
+			"description": "Sets the time after which relay 1 turns off once it has been turned on.",
+			"unit": "minutes",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "6",
+			"$if": "firmwareVersion >= 1.1",
+			"label": "Relay 1: Auto Turn-Off Timer",
+			"description": "Using the units defined in parameter 15. Sets the time after which relay 1 turns off once it has been turned on.",
```

Comment:
Nit: I'd switch these two sentences - likewise for the other parameters

Reply:
Fair enough, I had swapped it from the original file as I thought with the label, this was probably the more useful/important part of the description.